contribute-to-the-book.md

```diff
@@ -1,16 +1,16 @@
 # Contribute to the book
 
-Anyone is encouraged to add anything you want to this guide. Want to tell others how to compile a project written in Java? Want to provide a solution for a commonly seen issue? Or just want to let the document earlier to finish? This article will guide you to make contributions to this document.
+Everyone is encouraged to add anything they want to this guide. Want to tell others how to compile a project written in Java? Want to provide a solution for a commonly seen issue? Or just want to let the document earlier to finish? This article will guide you to make contributions to this document.
```

Comment:
they -> he

Reply:
Are you suggesting changing it from "they" to "he"?

I wouldn't recommend it because "they" refers to anyone who was last referenced, whereas "he" refers to a male who was last referenced. Since there are probably both males an females that participate, "they" makes more sense.

In the end, it is your choice.